app.rb

```diff
@@ -57,4 +61,16 @@ def run
     exit if answer == 7
     option_checker(answer)
   end
+
+  def load_data
+    load_people
+    puts
+    load_books
+    puts
+    load_rentals
```

Comment:
- [ ] Also, I encountered an error after fixing the pathname in the `person.rb` file as seen in the screenshot below

![Screenshot from 2022-08-12 06-07-25](https://user-images.githubusercontent.com/46388113/184296529-66687fcd-80ab-444a-9c69-e89b65425a9a.png)

Kindly fix it